subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/inputs/undeclared/UndeclaredBuildInputsIntegrationTest.groovy

```diff
@@ -592,6 +600,56 @@ class UndeclaredBuildInputsIntegrationTest extends AbstractConfigurationCacheInt
         outputContains("value = value")
     }
 
+    @Issue("https://github.com/gradle/gradle/issues/25044")
+    def "plugin can read file within jar"() {
```

Comment:
`Files.readString` is Java 11+ API, and Gradle is tested on Java 8 too. Could you please annotate the test with
`@Requires(UnitTestPreconditions.Jdk11OrLater)`?

Reply:
Thanks for the hint :thumbsup: